src/jit/lowerarm64.cpp

```diff
@@ -1370,7 +1370,6 @@ void Lowering::TreeNodeInfoInitBlockStore(GenTreeBlk* blkNode)
             // CopyBlk
             unsigned   size                  = blkNode->gtBlkSize;
             GenTreePtr dstAddr               = blkNode->Addr();
-            GenTreePtr srcAddr               = blkNode->Data();
```

Comment:
Note that there is code above this that sets `srcAddr`. This is an old (re)definition that was inadvertently left in.